app/registries/menu/loader.rb

```diff
@@ -74,7 +74,7 @@ def self.load
           menu.divider                  :caption => N_('Puppet')
           menu.item :environments,      :caption => N_('Environments')
           menu.item :puppetclasses,     :caption => N_('Classes')
-          menu.item :config_groups,     :caption => N_('Config Groups')
+          menu.item :puppetclass_lookup_keys, :caption => N_('Smart Class Parameters')
```

Comment:
This has been already dropped in previous commit in this branch, we should keep it removed :+1: 
```suggestion
          menu.item :puppetclass_lookup_keys, :caption => N_('Smart Class Parameters')
```

Reply:
done